CONTRIBUTING.md

```diff
@@ -0,0 +1,113 @@
+## Tests
+
+In a effort to cover as much of the codebase with tests as possible, new contributions
+should come with tests when possible/it makes sense.
+
+MDX has both unit tests and end-to-end tests.
+
+### Unit testing
+
+We should aim at improving the unit tests coverage as much as possible.
+Our unit tests can be found in [`test/lib/`](https://github.com/realworldocaml/mdx/tree/master/test/lib).
```

Comment:
This makes the link useless outside of github or when browsing a different branch than master or a commit.

```suggestion
Our unit tests can be found in [`test/lib/`](test/lib).
```
I'm not 100% sure this will work.

Reply:
It does work although I'm unsure whether it makes to this since the only place where it's supposed to work is on master. I'll update the links since it makes the md file more readable on its own!

Reply:
Should be good now!